app/src/main/java/com/example/healthlog/MainActivity.java

```diff
@@ -65,9 +65,7 @@ public void onClick(DialogInterface dialogInterface, int i) {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        MenuInflater inflater = getMenuInflater();
-        inflater.inflate(R.menu.menu_logout, menu);
```

Comment:
As you removed the logout button from here and added in each fragment but you didn't add the dialog function in each fragment because of that the button won't work so please fix that too.  Or try to call the dialog function on the button click

Reply:
But I checked on the app just now that logout button is working fine

Reply:
But I checked on the app just now that logout button is working fine

Reply:
can you upload a screen shot ?

Reply:
![Screenshot_20201213-085725.png](https://user-images.githubusercontent.com/58428413/102002311-4b365880-3d21-11eb-9a80-28c2c168d031.png)